src/main/resources/application.yml

```diff
@@ -132,6 +145,19 @@ nakadi:
       commitTimeout: 5 # seconds
   subscription:
     maxPartitions: 30
+  features.defaultFeatures:
```

Comment:
do we really need this duplication? I understand the purpose, make it visible for other.

Reply:
what is about this one ?